models/articles-model.js

```diff
@@ -2,7 +2,14 @@ const db = require('../db/connection');
 
 exports.fetchArticleByID = (article_id) => {
     return db
-    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .query(`
+    SELECT articles.article_id, title, topic, articles.author, articles.body, articles.created_at, articles.votes, COUNT(comments.author) AS comment_count
```

Comment:
Can just use SELECT articles.* as you're getting all the columns from articles table

Reply:
Sounds good thank you Scarlett! 😁